### DISCUSSION TITLE
Need my S3 Bucket mounted as a Drive.... Do I need MountainDuck?

### DISCUSSION THREAD

**Discussion by rebeccaliiz** [iterate-ch/cyberduck#15270] [Q&A] [answered]
My team needs to have an S3 bucket mounted as a Drive. This is because our users (scientists) need to import files into another clunky vendor-provided software in order to runt heir analysis, and then export back into the S3. These are scientists so not comfortable AWS/S3 users, hence why we need CyberDuck to mount the S3 bucket as a Drive.

Does CyberDuck provide this capability? Or is MountainDuck actually what I need?

**Comment by dkocher** [comment, marked as answer]
What you want is Mountain Duck.

> Mountain Duck lets you mount server and cloud storage as a disk in Finder on macOS and the File Explorer on Windows. Open remote files with any application and work like on a local volume.

**Reply by rebeccaliiz** [reply]
I thought so. Thank you!
